docs/asciidoc/back_matter.adoc

```diff
@@ -39,3 +37,94 @@ This spec defines a standard **Data Repository Service (DRS) API** (“the yello
+
+== Appendix: Background Notes on DRS URIs
+
+=== Design Motivation
+
+DRS URIs are aligned with the https://www.nature.com/articles/sdata201618[FAIR data principles] and the https://doi.org/10.1038/sdata.2018.2[Joint Declaration of Data Citation Principles] -- both hostname-based and compact identifier-based URIs provide globally unique, machine-resolvable, persistent identifiers for data.
+
+* We require all URIs to begin with `drs://` as a signal to humans and  systems consuming these URIs that the response they will ultimately receive, after transforming the URI to a fetchable URL, will be a DRS JSON packet. This signal differentiates DRS URIs from the wide variety of other entities (HTML documents, PDFs, ontology notes, etc.) that can be represented by compact identifiers.
+* We support hostname-based URIs because of their simplicity and efficiency for server and client implementers.
+* We support compact identifier-based URIs, and the meta-resolver services of identifiers.org and n2t.net (Name-to-Thing), because of the wide adoption of compact identifiers in the research community. as detailed by https://doi.org/10.1038/sdata.2018.29[Wimalaratne et al (2018)] in "Uniform resolution of compact identifiers for biomedical data."
+
+== Appendix: Compact Identifier-Based URIs
+
+.Note: Identifiers.org/n2t.net API Changes
+****
+The examples below show the current API interactions with https://n2t.net/e/compact_ids.html[n2t.net] and https://docs.identifiers.org/[identifiers.org] which may change over time.  Please refer to the documentation from each site for the most up-to-date information.  We will make best efforts to keep the DRS specification current but DRS clients MUST maintain their ability to use either the identifiers.org or n2t.net APIs to resolve compact identifier-based DRS URIs.
+****
+
+=== Registering a DRS Server on a Meta-Resolver
+
+See the documentation on the https://n2t.net/e/compact_ids.html[n2t.net] and https://docs.identifiers.org/[identifiers.org] meta-resolvers for adding your own compact identifier type and registering your DRS server as a resolver. You can register new prefixes (or mirrors by adding resource provider codes) for free using a simple online form.  Keep in mind, while anyone can register prefixes, the identifiers.org/n2t.net sites do basic hand curation to verify new prefix and resource (provider code) requests.  See those sites for more details on their security practices.
+
+Starting with the prefix for our new compact identifier, let's register the namespace `mydrsprefix` on identifiers.org/n2t.net and use 5-digit numeric IDs as our accessions.  We will then link this to the DRS server at `https://mydrs.server.org/ga4gh/drs/v1/` by filling in the provider details.  Here's what that the registration for our new namespace looks like on https://registry.identifiers.org/prefixregistrationrequest[identifiers.org]:
+
+image::prefix_register_1.png[]
+
+image::prefix_register_2.png[]
+
```

Comment:
Let's move this section to the "More background info" doc -- it adds a lot of clutter, is only relevant to a minority of readers, and is the most likely part to get outdated if meta-resolvers change.

Reply:
I think it's worth pointing out in the main spec but I agree that it adds clutter and, more importantly, something that may age very poorly.  So I left in the mention of this via the first paragraph but then put the actual example in the background doc as you suggested.